.github/workflows/checkoss.yaml

```diff
@@ -0,0 +1,16 @@
+name: checkoss
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: checkoss
+        run: |
+          scripts/dev/checkoss.sh
```

Comment:
wonder if it makes sense to put scripts used from workflows into scripts/workflow/ ?

Reply:
checkoss is also exposed through ninja...I am not completely sold on the current file organization at the moment. I kind of liked @beckerg's comments on mimicking some sort of Linux standard file system hierarchy a bit, although I would still keep all the source code as is in cli, lib, tests, tools, etc. Then all these scripts could just be thrown in `bin/` and be done with it.